control_plane/src/storage.rs

```diff
@@ -63,7 +63,7 @@ impl ResponseErrorMessageExt for Response {
         Err(PageserverHttpError::Response(
             match self.json::<HttpErrorBody>() {
                 Ok(err_body) => format!("Error: {}", err_body.msg),
-                Err(_) => format!("Http error ({}) at {}.", status.as_u16(), url),
+                Err(_) => format!("Http error ({}) at {url}.", status.as_u16()),
```

Comment:
I find it confusing to mix the `".. {}", var` and `"{var}"` styles in the same message, so I'd prefer to leave it as it is.